saleor/plugins/invoicing/utils.py

```diff
@@ -18,31 +18,30 @@
 MAX_PRODUCTS_PER_PAGE = 13
 
 
-def make_full_invoice_number(number=None, month=None, year=None):
+def make_full_invoice_number(number=None, year=None, begin_number=None):
     now = datetime.now()
-    current_month = int(now.strftime("%m"))
     current_year = int(now.strftime("%Y"))
-    month_and_year = now.strftime("%m/%Y")
-
-    if month == current_month and year == current_year:
+    if begin_number is not None:
```

Comment:
it's safer to write "if not begin_number" in case someone cleared begin_number in dashboard and empty string is passed